compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -5478,10 +5479,13 @@ private BLangRawTemplateLiteral createRawTemplateLiteral(NodeList<Node> members,
         return literal;
     }
 
-    private BLangNode createRegExpTemplateLiteral(NodeList<Node> reSequences, Location location) {
+    private BLangNode createRegExpTemplateLiteral(NodeList<Node> reSequences, Location location, Token startBackTick,
+                                                  Token endBackTick) {
         BLangRegExpTemplateLiteral regExpTemplateLiteral =
                 (BLangRegExpTemplateLiteral) TreeBuilder.createRegExpTemplateLiteralNode();
-        regExpTemplateLiteral.reDisjunction = (BLangReDisjunction) createReDisjunctionNode(reSequences, location);
+        Location reDisjunctionPos = getPosition(startBackTick, endBackTick);
```

Comment:
```suggestion
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39466/commits/7ae121096651ef8776fb470d5f56096e846c08e6.